apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -43,6 +43,21 @@
     }
   }
 
+  const Clipboard = Quill.import('modules/clipboard');
+  class PotentiallyDisabledClipboard extends Clipboard {
+    onCaptureCopy(e, isCut = false) {
+      if (this.options.enabled) return super.onCaptureCopy(e, isCut);
```

Comment:
Kind of nit, but may be a good idea to treat undefined as "enabled".
```suggestion
      if (this.options.enabled === false) return super.onCaptureCopy(e, isCut);
```
Same below.